sanity/program.js

```diff
@@ -33,6 +33,10 @@ function getProgram(actions) {
           '--from-block [blockNumber]',
           'In order to not get from 0 to latest, set the started block to get events. Decentraland starts at 4900000'
         )
+        .option(
+          '--blocks [blocks]',
+          'Number of block to be substracted to the latest block to be used as a from-block'
```

Comment:
I really don't understand this flag

- What's the usecase for this flag? isn't `from-block` enough?
- If we need it, why do we have two flags that do essentially the same thing? `from-block` and `blocks`. Also it's a bit weird that you can send both, what happens then? 
- I read [this](https://github.com/decentraland/marketplace/pull/899/files#diff-b5d8952feeb3d2427a3ee269c547387eR94) first and the name was a bit weird to me, I didn't understand what was going on before reading this description. It's kind of hard to name tho.

Reply:
>I read this first and the name was a bit weird to me, I didn't understand what was going on before reading this description. It's kind of hard to name tho.

Right, the name is not the best. I thought `back-blocks` too. Can you help me to think a better one?

> What's the usecase for this flag? isn't from-block enough?
> If we need it, why do we have two flags that do essentially the same thing? from-block and blocks. Also it's a bit weird that you can send both, what happens then?

No, `from-block` is designed to used when you want to sanitize from a specific block number and `blocks` is designed to be used when you want to sanitize from `latest - N` blocks. 
As we cron this process, we don't have the latest block number to do the math.

Reply:
Ok so I think it's like an offset then? It's hard to name because it's a complex concept inside a single flag

I'm still unsure what happens when you supply `from-block` and `blocks` and how are we going to run it. Are we going to use only `blocks`?
If so maybe `from-block-offset`? that way:

```
sanity # start from latest
sanity --from-block N # start from N
sanity --from-block N --from-block-offset M # start from N-M
sanity --from-block-offset M # start from latest-M
```

that seems consistent

Reply:
Using both at the same time is useless. I [prioritized `block`](https://github.com/decentraland/marketplace/pull/899/files#diff-b5d8952feeb3d2427a3ee269c547387eR91) in this case. 

But I think that the right behaviour should be prioritize the specific block you set.

`sanity # start from 0` (not latest, sanity from latest wont make sense)
`sanity --from-block N --from-block-offset M # start from N`